cli/cmd/event/search.go

```diff
@@ -40,14 +40,61 @@ func runSearchEvent(_ *cobra.Command, _ []string) error {
 		params["id"] = id
 	}
 
+	name := viper.GetString("name")
+	if name != "" {
+		params["name"] = name
+	}
+
+	version := viper.GetString("version")
+	if version != "" {
+		params["version"] = version
+	}
+
+	release := viper.GetString("release")
+	if release != "" {
+		params["release"] = release
+	}
+
+	platformID := viper.GetString("platform_id")
+	if platformID != "" {
+		params["platform_id"] = platformID
+	}
+
+	pkg := viper.GetString("package")
+	if pkg != "" {
+		params["package"] = pkg
+	}
+
+	success := viper.GetString("success")
+	if success != "" {
+		params["success"] = success
+	}
+
+	eventReceiverID := viper.GetString("event-receiver-id")
```

Comment:
Inconsistent with flag in `NewSearchCommand()` which uses "event_receiver_id" (underscores instead of hyphens). Now looking at it, I'd suggest hyphens be used for CLI flags. If changing receiver ID, also switch "platform_id".